apps/web/moon.yml

```diff
@@ -38,8 +38,8 @@ tasks:
     - concurrently
     - -n Next,Relay-Compiler,GraphQL-Codegen
     - yarn next dev
-    - yarn moon run web:relay-watch
-    - yarn moon run web:graphql-codegen-watch
+    - node_modules/.bin/moon run web:relay-watch
+    - node_modules/.bin/moon run web:graphql-codegen-watch
```

Comment:
the new yarn doesn't let you run binary files which is super annoying 

There's even a thread [here](https://github.com/yarnpkg/berry/issues/882) with Miles on it